contrib/mesos/pkg/executor/executor_test.go

```diff
@@ -543,7 +543,7 @@ func NewMockPodsListWatch(initialPodList api.PodList) *MockPodsListWatch {
 		list:        initialPodList,
 	}
 	lw.ListWatch = cache.ListWatch{
-		WatchFunc: func(resourceVersion string) (watch.Interface, error) {
+		WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
```

Comment:
I feel like a pointer would be more natural, but I can't think of a concrete reason to ask you to switch...